src/models/calendar-restful-model-collection.ts

```diff
@@ -100,49 +101,39 @@ export default class CalendarRestfulModelCollection extends RestfulModelCollecti
     options: ConsecutiveAvailabilityQuery,
     callback?: (error: Error | null, data?: { [key: string]: any }) => void
   ): Promise<CalendarConsecutiveAvailability> {
-    // If open hours contains any emails not present in the main emails key
-    // or in the free busy email list as this would raise an error on the API side
-    const freeBusyEmails = options.freeBusy
-      ? options.freeBusy.map(fb => fb.email)
-      : [];
-    if (options.openHours) {
-      for (const openHour of options.openHours) {
-        for (const email of openHour.emails) {
-          if (
-            !options.emails.some(row => row.includes(email)) &&
-            !freeBusyEmails.includes(email)
-          ) {
-            throw new Error(
-              'Open Hours cannot contain an email not present in the main email list or the free busy email list.'
-            );
+    if (!options.emails && !options.calendars) {
```

Comment:
1. that looks right, you need at least one but both are fine
2. how detailed do you plan on going for error handling at this level?

Reply:
@tbadun I'm just implementing simple client-side rules that reflect what the API would see as an error, to prevent the user from having to make a call that will 100% fail. I don't want to add error checking for things that are dependant on other parts of the API account (for example, I don't want to check if an account has the Calendar scope set before making a Calendar-related call - that can be done by the API).